README.md

```diff
@@ -1,24 +1,33 @@
 # Pallas
 
-An open source Solid State Interpreter exokernel and runtime
+An event sourced, purely functional operating system.
 
 ## Index
 
 1. [Introduction](#introduction)
-2. [Caveats](#caveats)
-3. [Installation](#installation)
-4. [Getting Started](#getting-started)
-5. [Example](#example)
-6. [Contributing](#contributing)
+2. [Installation](#installation)
+3. [Getting Started](#getting-started)
+4. [Example](#example)
+5. [Contributing](#contributing)
+6. [Caveats](#caveats)
 7. [Additional Resources](#additional-resources)
 
 ## Introduction
 
+Pallas is an event sourced, purely functional operating system, called an operating function. 
 
-Pallas is a purely functional exokernel and library OS, designed to radically simplify the modern networked computing stack. We call the libOS an "operating function" because it is defined as a pure function of its event input stream.
-The Pallas [SSI](https://wiki.vaporware.network/solid-state%20interpreter) programming environment is a virtual machine which is also a database engine. Databases are created by writing pure functions which fold their streams of incoming events, at each step returning an output event and possibly an upgraded version of the function itself.
+Pallas provides an entirely unique set of features, but is inspired by a long history of systems and language research, including MIT’s exokernel. Pallas collapses the distinction between database, virtual machine, and language platform, creating a stable, simple, and extensible application environment.
 
-Let's pick that apart. The foundation of Pallas is untyped, but conceptually we can say that a database is a function of the type `DB`:
+The cost to this design is a clean break with legacy systems, including Unix and most popular programming languages. In return, Pallas offers a set of features that are found nowhere else:
+- All application code is automatically persisted, without imports or boilerplate. To create a database, you write a pure function. System calls are included in persisted state.
+- Partially applied functions can be serialized and stored on-disk, or sent over the wire. Programs in mid-execution can be paused, moved to a new machine, and resumed with no impact.
+- Program execution can be parallelized via a process model. A single machine can spawn and manage multiple processes.
+- The system’s language blends features from both Lisp and Haskell, but uses a more readable and flexible syntax than S-expressions. Pallas supports metaprogramming, hot reload, and macro-based type systems. 
+- Data and code is deduplicated, merkleized, and stored in content-addressable memory pages. This creates a global referentially-transparent content store, which naturally complements protocols like BitTorrent.
+
+By reading our documentation and examples, you can prove to yourself that these claims are true.
```

Comment:
```suggestion
```

Reply:
Feels insecure? Maybe we should keep it but say something more like "There are [examples] and [docs] here, and if you want to talk to us we have a [Telegram]." Basically treat this as the end of the intro, and then launch into the more technical stuff below.